iron-ajax.html

```diff
@@ -118,7 +118,7 @@
        */
       contentType: {
         type: String,
-        value: 'application/x-www-form-urlencoded'
+        value: null
```

Comment:
That makes sense, but it looks like it prevents the `_requestOptionsChanged` observer from firing, because it's waiting for all arguments to be defined. As a result, the tests for `auto` fail.